src/database.rs

```diff
@@ -0,0 +1,63 @@
+extern crate rusqlite;
```

Comment:
So far, all the subprojects of Foxbox I've seen put `extern crate` declarations in `src/lib.rs` or `src/main.rs`, let's stick to this convention.